### PR TITLE
Install Go on Debian 10

### PR DESCRIPTION
**Pull request by Tumite**
I have read and understood the Contributor's Certificate of Origin available at the end of
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Roland Balla balla.roland96@gmail.com

### CODE REVIEW

File: tutorials/install-go-on-debian-10/01.en.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+## Step 1 - Downloading Go
+
+  
+First run the update command to update the package manager cache:
+
+ 
+```bash
+sudo apt update
+```
+  The current latest version of Golang available on the official website is `1.12.9` at the time writing this tutorial. You can get the current latest version URL by visiting their [download page](https://golang.org/dl/).
```
Comment (n-se) [inline tutorials/install-go-on-debian-10/01.en.md]: There is already go1.13, mind updating this before we merge it? :)

Comment (Tumite) [reply tutorials/install-go-on-debian-10/01.en.md]: Yes, of course. 1-2 day and i wil update.

Comment (Tumite) [reply tutorials/install-go-on-debian-10/01.en.md]: Updated.

Comment (n-se) [reply tutorials/install-go-on-debian-10/01.en.md]: Did you forgot to push the changes?

Comment (Tumite) [reply tutorials/install-go-on-debian-10/01.en.md]: No, i pushed the updated 1.13 version in my repo.

**Comment by leojakeson** [comment]
Hello Tumite, as with #186 this tutorial is too short and not detailed enough to be added to our Community Tutorials. On top of that tecadmin.net has a very similar tutorial.

This tutorial won't be accepted, sorry.

**Pull request closed without merging**